grails-app/controllers/au/org/ala/volunteer/AjaxController.groovy

```diff
@@ -703,4 +703,26 @@ class AjaxController {
             request.inputStream
         }
     }
+
+    def expeditionDwcJson() {
+        def project = Project.get(params.long('id'))
+        def result = [:]
+
+        if (project) {
+            def taskList = taskService.getAllTasksAndTranscriptionsIfExists(project, [max: 9999])
+            def fieldList = fieldService.getAllFieldsWithTasks(taskList)
+            def fieldNames =  ["taskID", "taskURL", "validationStatus", "transcriberID", "validatorID",
+                               "externalIdentifier", "exportComment", "dateTranscribed", "dateValidated"]
+            fieldNames.addAll(fieldList.name.unique().sort() as List<String>)
+            result.data = exportService.exportJson(taskList, fieldList)
+            result.projectId = project.id
+            result.institutionId = project.institution.id
+
+            result.success = true
+            render(result as JSON)
+        } else {
+            render([status: 404, message: "Expedition not found."] as JSON)
```

Comment:
```suggestion
            response.status = SC_NOT_FOUND
            render([message: "Expedition not found."] as JSON)
```

Setting the response status would be great, that differentiates responses without needing to parse the body